janus-frontend/src/components/LinkInput.tsx

```diff
@@ -1,5 +1,7 @@
-import React from "react";
+import React, { useState, useRef } from "react";
 import styled from "styled-components";
+import validator from "validator";
```

Comment:
Why add a 28kb library for something that a regex pattern can do?

https://bundlephobia.com/package/validator@13.7.0

Better to do this via regex